Content fixes pre release (#174)
bottlerocket-os/bottlerocket-project-website: includes/homepage/safe-update.markdown

{{</ stackedfigure>}}

Because system settings are accomplished through an API, Bottlerocket knows how to migrate these settings between versions.

> Should something go awry during an update, Bottlerocket has built-in logic that lets you revert to the previously working version with your settings intact.

Reviewer: ```suggestion
Should something go awry during an update, the Bottlerocket update system lets you revert to the previously working version with your settings intact.
```
